Sources/Core/Public/GTMSessionFetcher/GTMSessionFetcher.h

```diff
@@ -1084,9 +1085,12 @@ __deprecated_msg("implement GTMSessionFetcherAuthorizer instead")
 @property(atomic, readonly, getter=isFetching) BOOL fetching;
 
 // Cancel the fetch of the request that's currently in progress.  The completion handler
-// will not be called.
+// will be called if the property `stopFetchingTriggersCompletionHandler` is `YES`.
 - (void)stopFetching;
 
+// Call callbacks in `stopFetching`. It cannot be changed once the fetcher starts.
+@property(atomic, assign) BOOL stopFetchingTriggersCompletionHandler;
```

Comment:
Sorry, didn't think of this before – Can you add a block here speaking to the point if the fetcher is going to be use from Swift `async` calls, they this must be set to YES to ensure a cancel doesn't leave the Swift runtime in a bad state?.

Reply:
Done